### PR TITLE
PXB-2920 Fix misspelling in versioning scheme (2.4)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified: docs/xtrabackup-versioning-scheme.md
renamed: docs/xtrabackup-versioning-scheme.md -> docs/xtrabackup-version-numbers.md
modified: mkdocs-base.yml

### CODE REVIEW

File: docs/xtrabackup-versioning-scheme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### Percona XtraBackup 2.4 versioning scheme:
+# Percona XtraBackup 2.4 versioning scheme:
```
Comment (patrickbirch) [inline docs/xtrabackup-versioning-scheme.md]: Colon?

File: docs/xtrabackup-versioning-scheme.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 
 Here is what the numbers in the product version specification mean:
```
Comment (patrickbirch) [inline docs/xtrabackup-versioning-scheme.md]: See the PXC version. This sentence is out of place and shouldn't end with a colon.

File: docs/xtrabackup-versioning-scheme.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup-versioning-scheme.md]: updated

File: docs/xtrabackup-versioning-scheme.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup-versioning-scheme.md]: Updated

File: docs/xtrabackup-version-numbers.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Percona uses semantic version numbering, which follows the pattern of base version and minor version. Percona assigns unique, non-negative integers in increasing order for each version release.
+
+The version numbers for Percona XtraBackup 2.4.26 define the following information:
+
+* Base version - the leftmost number indicates the version of the Percona XtraBackup that is based on [MySQL 5.7](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/). A base version increase resets the minor version to 0. 
+
+* Minor version - an internal number that denotes the version of the software. A minor version increases by one each time the Percona XtraBackup is released.
+
+Percona does not release a new version of Percona XtraBackup 2.4 for every release of MySQL 5.7. Percona XtraBackup 2.4 is compatible with newer versions of MySQL 5.7.
```
Comment (patrickbirch) [inline docs/xtrabackup-version-numbers.md]: Move this paragraph to line 26.

Comment (alina-derkach-oaza) [reply docs/xtrabackup-version-numbers.md]: Updated

File: docs/xtrabackup-version-numbers.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        text-align: center;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+</style>
+
+| 2.4 | 26 |
+|---|---|
+| Base version | Minor |
```
Comment (patrickbirch) [inline docs/xtrabackup-version-numbers.md]: Bulld version instead of minor version
update everywhere.

Comment (alina-derkach-oaza) [reply docs/xtrabackup-version-numbers.md]: Updated

**Review by patrickbirch — APPROVED** [review]

**Pull request merged**